### PR TITLE
#1171 do not allow emojis with unicode version larger than 9.0

### PR DESCRIPTION
**Pull request by cebr76**

### CODE REVIEW

File: src/main/java/ai/elimu/web/content/emoji/EmojiCreateController.java [hunk 1]
```diff
@@ -44,6 +44,10 @@ public String handleSubmit(
         if (existingEmoji != null) {
             result.rejectValue("glyph", "NonUnique");
         }
+
+        if(emoji.getUnicodeVersion() > 9){
```
Comment (nya-elimuai) [inline src/main/java/ai/elimu/web/content/emoji/EmojiCreateController.java:48]: Add spacing before/after parenthesis: `if (emoji.getUnicodeVersion() > 9) {`

File: src/main/webapp/WEB-INF/i18n/errors_en.properties [hunk 2]
```diff
@@ -29,3 +29,4 @@ typeMismatch.thumbnail=Valid thumbnail format: PNG
 typeMismatch.java.lang.Integer={0} must be a number
 formatHint.Integer=Number (e.g. "5000")
 image.too.small=The image width must be at least 640px
+emoji.unicode.version=only emojis up to unicode version 9
```
Comment (nya-elimuai) [inline src/main/webapp/WEB-INF/i18n/errors_en.properties:32]: Change from lower-case to upper-case: "Only"

**Review by nya-elimuai — CHANGES_REQUESTED** [review]

**Review by nya-elimuai — CHANGES_REQUESTED** [review]
@cebr76 Looks great, thank you 🙂

Can you also add this code modification to the `EmojiEditController`?

**Review by nya-elimuai — APPROVED** [review]

File: src/main/webapp/WEB-INF/i18n/errors_en.properties [hunk 3]
```diff
@@ -29,3 +29,4 @@ typeMismatch.thumbnail=Valid thumbnail format: PNG
 typeMismatch.java.lang.Integer={0} must be a number
 formatHint.Integer=Number (e.g. "5000")
 image.too.small=The image width must be at least 640px
+emoji.unicode.version=Only emojis up to unicode version 9
```
Comment (jo-elimuai) [inline src/main/webapp/WEB-INF/i18n/errors_en.properties:32]:
```suggestion
emoji.unicode.version=Only emojis up to Unicode version 9
```

**Review by jo-elimuai — APPROVED** [review]

**Review by nya-elimuai — APPROVED** [review]

**Comment by nya-elimuai** [comment]
@cebr76 Your changes have been incorporated into the `master` branch. Thank you for contributing 🙂

**Pull request merged**